app/src/main/java/at/bitfire/icsdroid/model/CredentialsModel.kt

```diff
@@ -6,8 +6,8 @@ import at.bitfire.icsdroid.db.entity.Credential
 
 class CredentialsModel : ViewModel() {
     val requiresAuth = MutableLiveData(false)
-    val username = MutableLiveData("")
-    val password = MutableLiveData("")
+    val username = MutableLiveData<String>(null)
+    val password = MutableLiveData<String>(null)
```

Comment:
If we are initializing as `null`, and in fact, the values can be null, the value should be nullable:
```kotlin
    val username = MutableLiveData<String?>(null)
    val password = MutableLiveData<String?>(null)
```

Reply:
true that!